packages/office-ui-fabric-react/src/utilities/rtl.test.ts

```diff
@@ -0,0 +1,23 @@
+import { createTheme, ITheme } from '@uifabric/styling';
+import { getRTLWithTheme } from './rtl';
+
+describe('getRTLWithTheme', () => {
+  it('returns document default (ltr) when called with no theme', () => {
+    expect(getRTLWithTheme()).toBeFalsy();
+  });
+
+  it('returns document default (ltr) when called theme not specifying direction', () => {
+    const theme: ITheme = createTheme({});
+    expect(getRTLWithTheme(theme)).toBeFalsy();
+  });
+
+  it('returns ltr when called theme specifying ltr', () => {
+    const theme: ITheme = { ...createTheme({}), rtl: false };
+    expect(getRTLWithTheme(theme)).toBeFalsy();
+  });
+
+  it('returns rtl when called theme specifying rtl', () => {
+    const theme: ITheme = { ...createTheme({}), rtl: true };
+    expect(getRTLWithTheme(theme)).toBeTruthy();
+  });
+});
```

Comment:
You could have `const theme = createTheme({})` on top and use that instead of calling `createTheme` in every test.

Reply:
Plus one for using beforeEach and afterEach

Reply:
It's minor, but it does feel like a tradeoff. The benefits of moving the theme creation up in context saves a few milliseconds, but in my opinion it introduces a few risks:
- tests become DRY instead of [DAMP](http://blog.jayfields.com/2006/05/dry-code-damp-dsls.html)
- easy for someone to accidentally modify the `theme` object